arborcat.module

```diff
@@ -1,120 +1,216 @@
 <?php
+
+use Drupal\Core\Datetime\DrupalDateTime;
+
 /**
  * Display help and module information
  * @return help text for section
  */
-function arborcat_help($path, $arg) {
-  $output = '';
+function arborcat_help($path, $arg)
+{
+    $output = '';
 
-  switch ($path) {
+    switch ($path) {
     case "admin/help#arborcat":
       $output = '<p>' .  t("Catalog functions for Evergreen") . '</p>';
       break;
   }
 
-  return $output;
+    return $output;
 }
 
-function arborcat_theme($existing, $type, $theme, $path) {
-  return [
+function arborcat_theme($existing, $type, $theme, $path)
+{
+    return [
     'catalog' => [
       'variables' => [
-        'catalog_slider' => NULL,
-        'community_slider' => NULL,
-        'podcast_slider' => NULL
+        'catalog_slider' => null,
```

Comment:
Drupal coding standards are different than Laravel, constants are supposed to be in all caps.
https://www.drupal.org/docs/develop/standards/coding-standards#s-constants